app/pages/index.tsx

```diff
@@ -116,8 +116,15 @@ const WhatsNew = styled.div`
     font-size: 20px;
     weight: 700;
   }
+  a {
+    color: #ffffff;
+    weight: 700;
+  }
 `;
 
+const technicalURL = 'https://bcgov.github.io/sso-docs/';
+const businessURL = 'https://mvp.developer.gov.bc.ca/docs/default/component/css-docs';
```

Comment:
For these two links, I added the baseURLs for them into utils/constants.ts that you can import and reuse, I am expecting these urls to change again, particularily the mvp.developer.gov.bc.ca.